ProcessMaker/Http/Controllers/RequestController.php

```diff
@@ -64,12 +64,16 @@ public function index($type = null)
         $userFilter = SaveSession::getConfigFilter('requestFilter', Auth::user());
 
         // Get default Saved search config
-        $defaultSavedSearch = SavedSearch::firstSystemSearchFor(
-            Auth::user(),
-            SavedSearch::KEY_REQUESTS,
-        );
-        if ($defaultSavedSearch) {
-            $defaultColumns = SavedSearchController::adjustColumnsOf($defaultSavedSearch->columns);
+        if (!class_exists(SavedSearch::class)) {
+            $defaultSavedSearch = SavedSearch::firstSystemSearchFor(
```

Comment:
When package is not installed is calling to `SavedSearch::firstSystemSearchFor` method, that method is in the package